osis_profile/models/education.py

```diff
@@ -0,0 +1,185 @@
+# ##############################################################################
+#
+#  OSIS stands for Open Student Information System. It's an application
+#  designed to manage the core business of higher education institutions,
+#  such as universities, faculties, institutes and professional schools.
+#  The core business involves the administration of students, teachers,
+#  courses, programs and so on.
+#
+#  Copyright (C) 2015-2021 Université catholique de Louvain (http://www.uclouvain.be)
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  A copy of this license - GNU General Public License - is available
+#  at the root of the source code of this program.  If not,
+#  see http://www.gnu.org/licenses/.
+#
+# ##############################################################################
+from django.db import models
+from django.utils.translation import gettext as _
+
+from base.models.person import Person
+from osis_profile.models.enums.education import (
+    HighSchoolDiplomaTypes,
+    DiplomaResults,
+    BelgianCommunitiesOfEducation,
+    EducationalTransition,
+    EducationalQualification,
+    ForeignDiplomaTypes,
+)
+
+
+class Institute(models.Model):
+    community = models.CharField(
+        _("Community"),
+        max_length=25,
+        null=True,
+    )
+    postal_code = models.CharField(
+        _("Postal code"),
+        max_length=25,
+        null=True,
+    )
+    name = models.CharField(
+        _("Institute"),
+        max_length=25,
+        null=True,
+    )
```

Comment:
Pour moi ce modèle peut être "inliné" dans BelgianHighSchoolDiploma car il ne peut avoir plusieurs valeurs (ça serait au pire donc un one-to-one).

Autre point, à confirmer, mais on aurait peut-être une référence direct à l'établissement, donc pas besoin d'enregistrer le code postal ni la commune